<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Firewall</class>
 <widget class="QWidget" name="Firewall">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>591</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>591</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>801</width>
    <height>591</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>个人防火墙</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="infoBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="firewall.qrc">
      <normaloff>:/images/overview.png</normaloff>:/images/overview.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>85</width>
      <height>85</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ruleBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="firewall.qrc">
      <normaloff>:/images/rule.png</normaloff>:/images/rule.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>85</width>
      <height>85</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="logBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="firewall.qrc">
      <normaloff>:/images/log.png</normaloff>:/images/log.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>2</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>2</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>37</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="firewall.qrc">
     <normaloff>:/images/close3.png</normaloff>:/images/close3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="firewall.qrc">:/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>761</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>37</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="firewall.qrc">
     <normaloff>:/images/minimize.png</normaloff>:/images/minimize.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="menuBtn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>37</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="firewall.qrc">
     <normaloff>:/images/skin.png</normaloff>:/images/skin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>21</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="firewall.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_exitBtn_clicked()</slot>
 </slots>
</ui>
